exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchExporterConfiguration.java

```diff
@@ -113,6 +116,22 @@ public String toString() {
     }
   }
 
+  public static class AuthenticationConfiguration {
```

Comment:
this has to be documented in our default configuration otherwise a user will never find it, add it here https://github.com/zeebe-io/zeebe/blob/1984-es-auth/dist/src/main/config/zeebe.cfg.toml#L319-L347